sdcm/utils/common.py

```diff
@@ -303,14 +303,20 @@ def list_logs_by_test_id(test_id):
         return results
 
     def convert_to_date(date_str):
-        try:
-            t = datetime.datetime.strptime(date_str, "%Y%m%d_%H%M%S")  # pylint: disable=invalid-name
-        except ValueError:
+        time_formats = [
```

Comment:
1) why do you ignore pylint? it says the name is not according to the standards instead of changing the name you shut up pylint....
2)  you can do simpler:
```
def convert_to_date(date_str):
	for time_format in [ "%Y%m%d_%H%M%S", "%Y_%m_%d_%H_%M_%S"]:
		try:
			return datetime.datetime.strptime(date_str, time_format)
		except ValueError:
			continue
	return datetime.datetime.now()
```

Reply:
2. didn't want to for time_format in [ "%Y%m%d_%H%M%S", "%Y_%m_%d_%H_%M_%S"] in case if formats will be added in future, i thought it will be easier to add to list structure, than update the loop.
regarding now(), it will not exactly now, this is done for testruns which will not have a date_time in path, what means that this is old log data. That is why mark them with old data_time